zerver/views/user_settings.py

```diff
@@ -269,6 +270,10 @@ def json_change_settings(
         except ValidationError as e:
             raise JsonableError(e.message)
 
+        ratelimited, _ = RateLimitedUser(user_profile, domain="email_change_by_user").rate_limit()
```

Comment:
s/_/ignored/ fixes the linter error; I'll deal.

Reply:
Pushed a version changing this.  (The issue is that `_`  is used for i18n in Django).